wouso/games/teamquest/models.py

```diff
@@ -2,19 +2,26 @@
 
 from wouso.core.game.models import Game
 from wouso.core.user.models import Player, PlayerGroup
+from wouso.core.qpool.models import Question
 
 
 class TeamQuestUser(Player):
-    group = models.ForeignKey('TeamQuestGroup', null=True, blank=True, default=None, related_name='users')
-	
-    def is_head(self):
-        if self.group is None:
-            return False
-        return self.group.head == self
+    pass
 
 
 class TeamQuest(models.Model):
-	pass
+    QUEST_TYPES = (
```

Comment:
We should not have types for now. I think all the answers should be "In site text answers"